### ISSUE TITLE
Drag and drop moving the file will not override no matter the confirmation

### ISSUE THREAD

**Issue by alpracka** [atom/tree-view#1365]
### Description

Drag and drop moving the file will not override no matter the confirmation

![Gif video](https://timi.cz/system/atom-tree-view-bug.gif)

### Steps to Reproduce

1. Drag and move file with the same filename to another folder
2. Drop the file into another folder
3. Confirm the dialog by clicking the button "Replace file" or hitting the enter
4. The file was not moved nor override

**Expected behavior:**

The file should be moved and override.

**Actual behavior:**

The file is not moved nor override.

**Reproduces how often:**

100%

### Versions

Atom    : 1.52.0
Electron: 6.1.12
Chrome  : 76.0.3809.146
Node    : 12.4.0

apm  2.5.2
npm  6.14.8
node 12.4.0 x64
atom 1.52.0
python 2.7.17
git 2.29.1

macOS 10.15.6 (19G2021) Catalina

**Comment by aminya** [comment]
Does this happen in the latest version of the Atom? There was an issue with the prompts because of the API change of Electron but I think has been fixed.

**Comment by alpracka** [comment]
The issue still remains in the latest stable version 1.53.0, but **is fixed** in 1.54.0-beta0.

``` shell
$ apm --version
apm  2.5.2
npm  6.14.8
node 12.4.0 x64
atom 1.53.0
python 2.7.17
git 2.29.2
```
I've updated the macOS to 11.1 (20C69) meanwhile.
